### ISSUE TITLE
Update help page for updating passbolt

### ISSUE THREAD

**Issue by binaryfigments** [passbolt/passbolt_api#78]
I couldn't find the documentation repo for passbolt on github.

Maybe you want to change the update page:
https://www.passbolt.com/help/tech/update

And add:
```
git stash
git pull
git stash pop
```
So the configuration files get stored while doing a git pull.

**Comment by stripthis** [comment]
HI @binaryfigments, 

Thank you for your suggestion.

Unless the developers make a mistake this step should not be necessary. We aimed that all the configuration are not be tracked on git and to provide default files like: app.php.default, config.php.default. 

However in the past we made mistakes and have had issues specifically with email.php configuration as it is tracked by cakephp by default (e.g. it was removed and added back by mistake), so in this context your suggestion would have made sense.

Long story short, that shouldn't be necessary. Feel free to reopen if I misunderstood your proposal.
